collections/_posts/2023-01-04-Transfer-Learning.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: review
+title: "What is being transferred in transfer learning?"
+tags: pre-training transfer-learning large-scale
+author: "Julia Puig"
+cite:
+    authors: "Behnam Neyshabur, Hanie Sedghi and Chiyuan Zhang"
+    title:   "What is being transferred in transfer learning?"
+    venue:   "NeurIPS 2020"
+pdf: "https://arxiv.org/abs/2008.11687"
+---
+
+<br/>
+
+# Context
+
+In cases where we lack of a large amount of data or we want a fast training, we are interested in pre-trained networks that can generalize well from a source domain to a target domain.
+
+**Transfer learning** consists in training a model on source data and then fine-tune it in target data, and has encountered a big success in several applications. Still, it is not clear:
+
+* What is needed for a successful transfer leaning?
+* Which parts of the network make transfer learning work?
+
+In this paper, the authors try to answer to these questions. Their code is available [here](https://github.com/google-research/understanding-transfer-learning).
+
+<br/>
+
+# Experimental Setup
+* Data:
+	* Source data for pre-training: ImageNet.
+	* Target data for downstream tasks: CheXpert (X-ray) and DomainNet (real, clipart and quickdraw).
+
+
+* Models:
+	* Architecture: ResNet-50.
+	* **P-T**: pre-trained model then fine-tuned on the target domain.
+	* **RI-T**: random initialization model trained on the target domain.
+
+<br/>
+
+# Experiments
+#### 1. Role of feature reuse in transfer learning
+**Intuition**: reusing the pre-trained feature hierarchy explains the success of transfer learning. Therefore, we expect better performances when doing transfer learning on target domains that share similar visual features with the source domain. 
+
+![](/collections/images/transfer/transfer_learning_fig2.jpg)
+
+**However**, even distant target domains present enhanced performances with transfer learning. Are some other factors other than feature reuse being useful for the target domain?
+
+![](/collections/images/transfer/transfer_learning_fig1.jpg)
+
+![](/collections/images/transfer/transfer_learning_fig3.jpg)
+
+**Conclusion**:
+
+* Feature reuse is paramount in transfer learning.
+* Low-level statistics also boost the performance of transfer learning and in particular training speed.
+* P-T models converge much faster than RI-T ones.
+
+<br/>
+
+#### 2. What are the differences between P-T and RI-T models?
+By looking at the mistakes made by each model:
+ 
+* Samples for which P-T is incorrect and RI-T is correct are mostly hard samples.
+* Samples for which P-T is correct and RI-T is incorrect are mostly easy samples.
```

Comment:
I'm not sure to understand what is the interpretation of those two first points

Reply:
It refers to ambiguous and not-that-ambiguous samples, in the context of image classification. I modified it, as "hard" is indeed too general. Also, these observations are made by visually looking at the results, there is no metric involved